android-lib/src/main/java/com/kinvey/android/store/DataStore.java

```diff
@@ -442,6 +447,9 @@ public void find(KinveyListCallback<T> callback, KinveyCachedClientCallback<List
         new AsyncRequest<List<T>>(this, methodMap.get(KEY_GET_ALL), callback, getWrappedCacheCallback(cachedCallback)).execute();
     }
 
+    public void findCount(KinveyCountCallback callback) {
```

Comment:
Why `findCount`? Our other SDKs use `count`.

Reply:
Good point, I'll change to `count`.